### PR TITLE
avoid notice during fetch phase

### PR DESCRIPTION
**Pull request by lucatume**
Tickets:

* https://central.tri.be/issues/69542
* https://central.tri.be/issues/69824

Fixes a notice popping up during the "fetch" phase of the import that would generate a PHP error.

### CODE REVIEW

File: src/Tribe/Aggregator/Record/Queue_Processor.php [hunk 1]
```diff
@@ -191,7 +191,10 @@ protected function do_processing() {
 
 		$this->current_queue->set_in_progress_flag();
 		$processed = $this->current_queue->process( self::$batch_size );
-		$this->processed += $processed->activity->count( $this->current_queue->get_queue_type() );
+		// in the 'fetch' phase this will not be a Queue object
+		if ( is_a( $processed, 'Tribe__Events__Aggregator__Record__Queue' ) ) {
```
Comment (bordoni) [inline src/Tribe/Aggregator/Record/Queue_Processor.php]: Avoid using `is_a`, use instanceof.

Comment (lucatume) [inline src/Tribe/Aggregator/Record/Queue_Processor.php]: Why is that? Any specific reason?

**Comment by lucatume** [comment]
This should be ok

**Comment by bordoni** [comment]
👍

**Pull request merged**